CHANGELOG.md

```diff
@@ -7,11 +7,11 @@ In case, there is no issue present, the pull request implementing the feature is
 
 Note that this project **does not** adhere to [Semantic Versioning](https://semver.org/).
 
-## [Unreleased]
+## [Unreleased] – 2023-10-22
```

Comment:
don't worry about this, there is another fix for this

Reply:
Thanks for the reminder, I'm just worried why I can't pass the Markdown style check, but I guess for now it I don't need to do anything about it?

Reply:
> don't worry about this, there is another fix for this

Thanks for the reminder, I'm just worried why I can't pass the Markdown style check, but I guess for now it I don't need to do anything about it?

Reply:
You need to worry a bit. The issue is quite clear:

     10:1  error  Unexpected additional part: ' – 2023-10-22'  entry-for-every-versions

An unreleased version has no release date - but you added one ^^. Just remove it again.

Please also merge latest `main`, which improved the CHANGELOG.md check - https://github.com/JabRef/jabref/pull/10537